kannon/master.py

```diff
@@ -118,7 +118,7 @@ def _exec_bullet_task(self, task: TaskOnBullet) -> None:
         pkl_path = self._gen_pkl_path(task)
         make_target(pkl_path).dump(task)
         # Run on child job
-        job_name = gen_job_name(f"{self.job_prefix}-{task.get_task_family()}")
```

Comment:
[should] use `job_prefix` or delete `job_prefix` argument of `Kannon.__init__` . I think to use `job_prefix` is better.

Reply:
Thank you, 
Added a change at commit 6a93fd2.